ui-tests/common/com.liferay.ide.ui.swtbot/src/com/liferay/ide/ui/swtbot/condition/NoRunningProjectBuildingJobs.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.ui.swtbot.condition;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.core.runtime.QualifiedName;
+import org.eclipse.core.runtime.jobs.IJobManager;
+import org.eclipse.core.runtime.jobs.Job;
+
+/**
+ * @author Charles Wu
+ */
+public class NoRunningProjectBuildingJobs extends NoRunningJobsCondition {
+
+	public NoRunningProjectBuildingJobs() {
+	}
+
+	@Override
+	protected List<Job> checkRunningJobs() {
+		List<Job> runningJobs = new ArrayList<>();
+
+		IJobManager jobManager = Job.getJobManager();
+
+		Job[] jobs = jobManager.find(family);
+
+		for (Job job : jobs) {
+			String jobName = job.getName();
+
+			if (job.getProperty(_liferayProjectJob) != null ||
+				jobName.equals("Synchronize Gradle projects with workspace") ||
```

Comment:
Lets use Job properties instead of job names.  Those could change in the future.

Reply:
@Camork can you make a new change to do this? thanks.

Reply:
I will deal with it later.